docs/toml.md

```diff
@@ -141,7 +141,9 @@
 email = "test@traefik.io"
 
 # File used for certificates storage.
-# WARNING, if you use Traefik in Docker, don't forget to mount this file as a volume.
+# WARNING, if you use Traefik in Docker, don't forget to mount the folder containing this file as a volume.
+# storageFile = "/etc/traefik/acme/acme.json"
+# $ docker run -v "/my/host/acme:/etc/traefik/acme" traefik
```

Comment:
because when I've mount the file, it created a folder named acme.json on my host. 

If it's not an expected behavior, I can test on a fresh environment.